### ISSUE TITLE
Errors in usage.md and error reported by CodeChecker command

### ISSUE THREAD

**Issue by rdanter** [Ericsson/codechecker#3641]
**Describe the bug**
In usage.md in Step 2 it shows the following command to get a list of profiles:

CodeChecker checkers --profile list --details

but when you run this you get an error as follows:

$ CodeChecker checkers --profile list --details
[ERROR 2022-03-30 13:27] - Checker profile 'list' does not exist!
[ERROR 2022-03-30 13:27] - To list available profiles, use '--profile list'.

Note that the error message also says to use '--profile list' which is also incorrect.

**CodeChecker version**
$ CodeChecker version
[INFO 2022-03-30 13:48] - CodeChecker analyzer version:
---------------------------------------------------------------
Kind                 | Version                                 
---------------------------------------------------------------
Base package version | 6.19.1                                  
Package build date   | 2022-03-18T16:12                        
Git commit ID (hash) | 9fa0f9817f6e91eda2a9202e0a3bf5006accdc8c
Git tag information  | 6.19.1                                  
---------------------------------------------------------------

[INFO 2022-03-30 13:48] - CodeChecker web version:
------------------------------------------------------------------------------
Kind                                | Version                                 
------------------------------------------------------------------------------
Base package version                | 6.19.1                                  
Package build date                  | 2022-03-18T16:12                        
Git commit ID (hash)                | 9fa0f9817f6e91eda2a9202e0a3bf5006accdc8c
Git tag information                 | 6.19.1                                  
Server supported Thrift API version | 6.47                                    
Client Thrift API version           | 6.47                                    
------------------------------------------------------------------------------

(installed today via pip)

**To Reproduce**
See description.

**Expected behaviour**
Docs and error messages should be correct.

**Desktop (please complete the following information)**
 - OS: Linux

**Additional context**
n/a

**Comment by whisperity** [comment]
Yeah it seems something changed here. The `--help` for the command itself does say that "no value" passed to `--profile` lists the profiles:

>   --profile [PROFILE]   List checkers enabled by the selected profile. If no argument is given then available profiles are listed.

```
╰─ CodeChecker checkers --profile
 default
 sensitive
 security
 portability
 extreme
```
